hbase_storage/hbase1.0.0.x/src/main/java/com/splicemachine/access/hbase/HBaseConnectionFactory.java

```diff
@@ -243,6 +243,17 @@ public boolean createSpliceHBaseTables(){
         }
     }
 
+    public void createRestoreTable() {
```

Comment:
I would create the table unconditionally during upgrade, not during backup/restore.

Reply:
It is only necessary for a restored database.